docs/designs/client_tool.md

```diff
@@ -53,7 +53,7 @@ this command-line client tool.
     elasticdl zoo build --image=a_docker_registry/bright/elasticdl-wnd:1.0 .
     ```
 
-1. Push the Docker image to a remote registry (optional)
+1. Push the Docker image to a remote registry.
```

Comment:
For the execution in minikube，we may not need push the image to a remote registry.

Reply:
Or add a comment stating that " not needed if using Minikube locally"

Reply:
> Or add a comment stating that " not needed if using Minikube locally"

Good idea. Let's merge this PR first and add the comment statement in the next PR.